crates/node/src/manager.rs

```diff
@@ -83,6 +84,8 @@ pub struct EpochManager<P> {
     /// and returns the ack that the first epoch has started. At this point,
     /// the node manager has other tasks to await and will run normally.
     node_ready: Option<oneshot::Sender<()>>,
+    /// Output channel for consensus blocks to be executed.
+    consensus_output: broadcast::Sender<ConsensusOutput>,
```

Comment:
I see why you did this but don't love channels living outside the consensus bus.  One of its uses is to make it simple to find code that send and receives a channel and this violates that.  Probably no way around it, but might be worth noting it should only be accesses via the consensus bus.